swagger.json

```diff
@@ -1118,6 +1118,58 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/log_edits": {
+            "get": {
+              "tags": [
+                "Fleet"
+              ],
+              "summary": "/fleet/drivers/{driver_id:[0-9]+}/log_edits",
+              "description": "Get summary of log edits for a specified driver.",
```

Comment:
We are only returning carrier edits here. Should we describe that either in the route or description?

Reply:
how about `Get a summary of carrier edits for a specified driver`

would you also prefer the endpoint name `/carrier_edit`?